src/components/ApplicationGroup/Details/EnvCDDetails/EnvCDDetails.tsx

```diff
@@ -183,6 +176,7 @@ export default function EnvCDDetails({ filteredApps }: AppGroupDetailDefaultType
                 />
             )
         } else {
+            const selectedApp = pipelineList.find((envData) => envData.appId === +appId)
```

Comment:
Can we use the same pattern either `+` or `Number`?

Reply:
We can Use +